### PR TITLE
Issue#389

### PR DESCRIPTION
**Pull request by himanshuc3**
I will be doing commits to this Issue in small packaged commits.

### CODE REVIEW

**Comment by catarak** [comment]
sounds good! feel free to break down the original issue into smaller ones.

**Comment by catarak** [comment]
should i label this [WIP]?

**Comment by himanshuc3** [comment]
I think you should merge it! Helps me keep working on it. If otherwise, you think, it would be better to keep it in progress, it's fine that way too!

File: client/styles/components/_sidebar.scss [hunk 1]
```diff
@@ -17,6 +16,7 @@
 .sidebar__title {
   font-size: #{12 / $base-font-size}rem;
   display: inline-block;
+  color: #a8a8a8;
```
Comment (catarak) [inline client/styles/components/_sidebar.scss]: could you put this color into a variable and add it to the themes so it looks good in the light dark, and high contrast themes?

File: client/styles/components/_sidebar.scss [hunk 2]
```diff
@@ -58,7 +58,7 @@
   @include themify() {
     color: map-get($theme-map, 'inactive-text-color');
     &:hover > .file-item__content .sidebar__file-item-name {
-      color: map-get($theme-map, 'primary-text-color');
+      color: $p5js-pink;
```
Comment (catarak) [inline client/styles/components/_sidebar.scss]: could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?

File: client/styles/components/_sidebar.scss [hunk 3]
```diff
@@ -95,6 +95,10 @@
 .file-item__content {
   display: flex;
   position: relative;
+  padding-top: #{12 / $base-font-size}rem;
+  padding-bottom: #{12 / $base-font-size}rem;
+  border-top: 1px solid #f4f4f4;
```
Comment (catarak) [inline client/styles/components/_sidebar.scss]: could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?

File: client/styles/components/_sidebar.scss [hunk 4]
```diff
@@ -211,6 +215,8 @@
 
 .sidebar__icons {
   display: flex;
+  padding-top: 5px;
```
Comment (catarak) [inline client/styles/components/_sidebar.scss]: could you use `rem`s instead of pixels?

File: client/styles/components/_sidebar.scss [hunk 5]
```diff
@@ -291,3 +297,7 @@
 .sidebar__file-item--closed .file-item__children {
   display: none;
 }
+
+.file-item__children li:last-child{
```
Comment (catarak) [inline client/styles/components/_sidebar.scss]: could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?

**Comment by catarak** [comment]
going to close this PR since it seems to be stale. feel free to reopen if you're still working on it!

**Pull request closed without merging**